Certification standard/main.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>Cardano Certification Level Standards</title>
+    <script src='./respec-ccwg-commons.js' async class='remove'></script>
+  </head>
+
+  <body>
+    <section id='abstract'> 
+
+        This document defines the specification for certification of smart contracts on the Cardano blockchain. It outlines three different levels of certification which can be used to show that a smart contract went has been reviewed and was found to behave as expected and to not be vulnerable to a set of security weaknesses. 
```

Comment:
This is a strong claim as it stands. Is it really the case that all levels will show that it's not vulnerable to a particular security weakness, or is it the case that we provide evidence towards this claim? That would be the point about QuickCheck at least.